test/e2e/run_test.go

```diff
@@ -1047,4 +1047,25 @@ USER mail`
 		Expect(session.ExitCode()).To(Equal(0))
 		Expect(strings.Contains(session.OutputToString(), groupName)).To(BeTrue())
 	})
+
+	It("podman run --tz", func() {
+		session := podmanTest.Podman([]string{"run", "--tz", "foo", "--rm", ALPINE, "date"})
```

Comment:
Can we verify this inside the container as well, make sure the timezone has been correctly detected?

Reply:
The honolulu test should make sure the timezone has been correctly detected since it checks if the timezone is 
HST (hawaii standard time) while the input is Pacific/Honolulu. I'm not sure how to do verify the timezone inside the container though.